tools/docker/compose.md

```diff
@@ -79,13 +79,12 @@ web:
   ports:
    - "8000:8000"
   volumes:
-   - ./awesome_web:/code
   links:
    - db
 db:
   image: fedora-postgresql
   volumes:
-   - ./db:/var/lib/pgsql/data
+   - ./db:/var/lib/pgsql/data:z
```

Comment:
If I understand it correctly, this is for sharing between containers, right? _(`:z` -> correct)_

FYI: for sharing outside of container, you need capital `:Z`.